.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+      - id: check-added-large-files
+  - repo: https://github.com/psf/black
+    rev: 23.1a1
+    hooks:
+      - id: black
```

Comment:
@vasylboliuk please remove black hook, since we are going to use ruff which is faster, does same thing as black and is more flexible

Reply:
Done.